README.md

```diff
@@ -12,6 +12,18 @@ The TUM Campus App (TCA) is an open source project, developed by volunteers and
 
 It mostly targets phones, but can also be used on tablets or any other device that runs iOS. We also have a [mac app](https://github.com/TUM-Dev/Campus-iOS/releases/lastest). This is the repo for the iOS version of the TUM Campus App.
 
+
+## 🚨 Public Service Announcement 🚨
```

Comment:
Suggestion:

Currently, we are at a transitioning point for our TUM Campus Apps. We have three platform-specific versions: [iOS](https://github.com/TUM-Dev/Campus-iOS), [Android](https://github.com/TUM-Dev/Campus-Android), and [UWP](https://github.com/TUM-Dev/Campus-UWP). However, maintaining these distinct versions has proven challenging, with notable issues including feature disparity and the need for individual bug fixes for each platform. Furthermore, this model is dependent on having a dedicated maintainer for each app.

As such, we have decided to consolidate our resources and efforts towards the creation of a unified, cross-platform app using [Flutter](https://flutter.dev/). You can access the new codebase here: [github.com/TUM-Dev/Campus-Flutter](https://github.com/TUM-Dev/Campus-Flutter).

We invite and value your contributions to the ongoing development of the Campus-iOS. However, please note that we are transitioning towards a new Flutter-based codebase, expected to take over the platform-specific apps in 2024. 

By uniting our efforts under one platform, we aim to enhance our efficiency and streamline the feature sets across all versions, thereby offering a more cohesive and reliable user experience. We thank you for your understanding and look forward to your continued support during this exciting period of transition.

Reply:
Let's try to use one consistent text for all READMEs of the native TCAs :)